skimage/data/__init__.py

```diff
@@ -384,3 +385,46 @@ def stereo_motorcycle():
     return (load("motorcycle_left.png"),
             load("motorcycle_right.png"),
             _np.load(_os.path.join(data_dir, "motorcycle_disp.npz"))["arr_0"])
+
+
+def cbcl_database():
```

Comment:
Maybe rename to something more explicit? For example, `cbcl_faces`, `cbcl_face_database`?